torch/nn/modules/module.py

```diff
@@ -1765,6 +1766,7 @@ def named_buffers(self, prefix: str = '', recurse: bool = True) -> Iterator[Tupl
             recurse (bool): if True, then yields buffers of this module
                 and all submodules. Otherwise, yields only buffers that
                 are direct members of this module.
+            remove_duplicate: whether to remove the duplicated buffers in the result
```

Comment:
nit: needs type and default value as: `remove_duplicate (bool, optional): whether to remove the duplicated buffers in the result. Defaults to True.`
The arguments above should be updates as well.

Reply:
since it requires updating not just oss apis (some internal apis are using these args as well), I'll put a TODO for the other arg for now

Reply:
OK, only found to occurrences actually, I'll fix that in this PR then